internal/security/secretstore/init.go

```diff
@@ -421,73 +421,76 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
+		msgBusCredentials, err = getCredential(internal.BootstrapMessageBusServiceKey, secretStore, messagebusSecretName)
 		if err != nil {
 			if err != errNotFound {
-				lc.Error("failed to determine if MQTT credentials already exist or not: %w", err)
+				lc.Errorf("failed to determine if %s credentials already exist or not: %w", configuration.SecureMessageBus.Type, err)
 				return false
 			}
 
-			lc.Info("Generating new password for MQTT bus")
-			mqttPassword, err := secretStore.GeneratePassword(ctx)
+			lc.Infof("Generating new password for %s bus", configuration.SecureMessageBus.Type)
+			msgBusPassword, err := secretStore.GeneratePassword(ctx)
 			if err != nil {
-				lc.Error("failed to generate password for mqtt-bus")
+				lc.Errorf("failed to generate password for %s", messagebusSecretName)
 				return false
 			}
 
-			mqttBusCredentials = UserPasswordPair{
-				User:     defaultMqttUser,
-				Password: mqttPassword,
+			msgBusCredentials = UserPasswordPair{
+				User:     defaultMsgBusUser,
+				Password: msgBusPassword,
 			}
 		} else {
-			lc.Info("MQTT credentials already exist, skipping generating new password")
+			lc.Infof("%s credentials already exist, skipping generating new password", configuration.SecureMessageBus.Type)
 		}
 
-		lc.Infof("adding any additional services using mqtt-bus for knownSecrets...")
-		services, ok := knownSecretsToAdd[mqttSecretName]
+		lc.Infof("adding any additional services using %s for knownSecrets...", configuration.SecureMessageBus.Type)
+		services, ok := knownSecretsToAdd[messagebusSecretName]
 		if ok {
 			for _, service := range services {
-				err = addServiceCredential(lc, mqttSecretName, secretStore, service, mqttBusCredentials)
+				err = addServiceCredential(lc, messagebusSecretName, secretStore, service, msgBusCredentials)
 				if err != nil {
 					lc.Error(err.Error())
 					return false
 				}
 			}
 		}
-		err = storeCredential(lc, "security-bootstrapper-mqtt", secretStore, mqttSecretName, mqttBusCredentials)
+		err = storeCredential(lc, internal.BootstrapMessageBusServiceKey, secretStore, messagebusSecretName, msgBusCredentials)
 		if err != nil {
 			lc.Error(err.Error())
 			return false
 		}
 	}
 
 	// determine the type of message bus
-	// TODO: change the value of type for "redis" to "redisdb" in EdgeX 3.0 as it is a breaking change
 	messageBusType := configuration.SecureMessageBus.Type
 	var creds UserPasswordPair
 	supportedSecureType := true
+	var secretName string
 	switch messageBusType {
-	case "redis":
-		messageBusType = redisSecretName
+	case redisSecureMessageBusType:
 		creds = redisCredentials
-	case mqttSecretName:
-		creds = mqttBusCredentials
+		secretName = redisSecretName
+	case mqttSecureMessageBusType:
+		creds = msgBusCredentials
+		secretName = messagebusSecretName
 	default:
 		supportedSecureType = false
-		lc.Warnf("secure message type is not redis or mqtt-bus")
+		lc.Warnf("secure message bus %s is not supported", messageBusType)
```

Comment:
Need tick marks incase type is blank.
```suggestion
		lc.Warnf("secure message bus type of '%s' is not supported", messageBusType)
```